frontend/src/App.js

```diff
@@ -75,7 +78,14 @@ function App() {
     );
   }
 
+  // does a user have permissions to write or approve reports?
+  // include write or approve, this would be true
+  // otherwise, it'll return false
+  // const startOnActivityReport = (currentUser) => currentUser.permissions.find((permission) => {
+  // permission.scopeId === 3 || permission.scopeId === 5});
+
```

Comment:
the idea is that this would un-commented when we are ready to have users default to the dashboard. Also, I wasn't sure what was happening to the <Home /> component